wynnset/tapestry-wp
# Locked Thumbnail and Image Selection

Closes #433 
Adds:
 - lockedImageURL property to node in both front and back end
 - locked-thumb pattern defs for each node in d3
 - tagging background-url based on locked condition in d3

Steps to test:
 - Go to the node modal, appearance and add a locked image thumbnail
 - Go to access and lock the node based on another, currently incomplete node in the tapestry
 - Exit modal and verify that the locked thumbnail is being displayed
 - Complete the "parent" node in the access dependency relationship and verify the thumbnail reverts to its previous state
